christian-galarza/lesson-1/car_app/src/main/Car.java

```diff
@@ -0,0 +1,15 @@
+package main;
+
+/**
+ * This class declare the variables.
+ *
+ *  @author Jose Christian Galarza Crespo
+ *  @version 1.0.
+ */
+class Car {
+    boolean statusEngine = false;
```

Comment:
As we talk in classes all this Fields should be private and each should have their access methods if necessary. And also please avoid to initialize the fields when declaring them use a constructor instead.